boards/arm/efm32gg_slwstk6121a/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2019 Interay Solutions B.V.
+# Copyright (c) 2019 Oane Kingma
+# Copyright (c) 2020 Thorvald Natvig
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
```

Comment:
It seems to me that including this is only necessary if CONFIG_ETH_GECKO is defined, see other efm32gg board.

Reply:
Good catch! Updated.